tenseal/tensealcontext.h

```diff
@@ -214,6 +225,13 @@ class TenSEALContext {
     */
     double _scale = -1;
 
+    /*
+    Switches for automatic relinearization, rescaling, and modulus switching
+    */
```

Comment:
Maybe in the future we can use a bitmask for keeping the internal flags, for momery improvements.

something like
```
enum {
  flag_auto_relin = 1 << 0,
  flag_auto_rescale = 1 << 1,
  flag_auto_mod_switch = 1 << 2,
}
 uint8_t _flags = flag_auto_mod_switch | flag_auto_rescale | flag_auto_relin;```

Reply:
Yeah better, I will work on this